src/shared-endpoints.ts

```diff
@@ -0,0 +1,137 @@
+import { GenericAPIResponse } from './util/requestUtils';
+import RequestWrapper from './util/requestWrapper';
+
+export default class SharedEndpoints {
+  // TODO: Is there a way to say that Base has to provide this?
+  protected requestWrapper: RequestWrapper;
+
+  /**
+   *
+   * Market Data Endpoints
+   *
+   */
+
+  getOrderBook(params: {
+    symbol: string;
+  }): GenericAPIResponse {
+    return this.requestWrapper.get('v2/public/orderBook/L2', params);
+  }
+
+  getTickers(params?: {
+    symbol?: string;
+  }): GenericAPIResponse {
+    return this.requestWrapper.get('v2/public/tickers', params);
+  }
+
+  getSymbols(): GenericAPIResponse {
+    return this.requestWrapper.get('v2/public/symbols');
+  }
+
+  getLiquidations(params: {
+    symbol: string;
+    from?: number;
+    limit?: number;
+    start_time?: number;
+    end_time?: number;
+  }): GenericAPIResponse {
+    return this.requestWrapper.get('v2/public/liq-records', params);
+  }
+
+  getOpenInterest(params: {
+    symbol: string;
+    period: string;
+    limit?: number;
+  }): GenericAPIResponse {
+    return this.requestWrapper.get('v2/public/open-interest', params);
+  }
+
+  getLatestBigDeal(params: {
+    symbol: string;
+    limit?: number;
+  }): GenericAPIResponse {
+    return this.requestWrapper.get('v2/public/big-deal', params);
+  }
+
+  getLongShortRatio(params: {
+    symbol: string;
+    period: string;
+    limit?: number;
+  }): GenericAPIResponse {
+    return this.requestWrapper.get('v2/public/account-ratio', params);
+  }
+
+  /**
+   *
+   * Account Data Endpoints
+   *
+   */
+
+  getApiKeyInfo(): GenericAPIResponse {
+    return this.requestWrapper.get('v2/private/account/api-key');
+  }
+
+  /**
+   *
+   * Wallet Data Endpoints
+   *
+   */
+
+  getWalletBalance(params: {
+    coin?: string;
+  }): GenericAPIResponse {
+    return this.requestWrapper.get('v2/private/wallet/balance', params)
+  }
+
+  getAssetExchangeRecords(params?: {
+    limit?: number;
+    from?: number;
+    direction?: string;
+  }): GenericAPIResponse {
+    return this.requestWrapper.get('v2/private/exchange-order/list', params);
+  }
```

Comment:
Silly request, can you move this to be after `getWithdrawRecords`? To be in line with bybit docs. Thanks.